src/components/header/header.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
```

Comment:
It would make sense to be consistent with the naming of the files. This one (Header) is lower-case, the Logo and Menu components are capitalized though

Reply:
I actually meant the file name, not the component name. It's common practice to name components capitalized. `const Header = () => {...` was fine.

Just rename the file to Header.js instead of header.js

Reply:
done.